src/Form/Form.js

```diff
@@ -0,0 +1,77 @@
+import React, { useState } from "react";
+import classes from "./Form.module.css";
+import firebase from "firebase/app";
+import "firebase/firestore";
```

Comment:
I don't believe you need to import `'firebase/firestore'` here, since it's already imported somewhere else. This is a side-effect import - which is a import that doesn't importing anything specifically to use, unlike `import foo from 'firebase/firestore'` for instance. That means it just needs to be ran once. Since it's imported in `firebase.js`, we don't need it here.

Now, if removing it causes any bugs, that's a bigger issue! However I don't think that will happen 🤓

Reply:
Just ran a test and no issues. Thus, REMOVED! :-)